docs/sources/flow/reference/stdlib/trim_suffix.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases:
+- ../../configuration-language/standard-library/trim_suffix/
+title: trim_suffix
+---
+
+# trim_suffix
+
+`trim_suffix` removes the specified suffix from the end of the given string.
```

Comment:
```suggestion
`trim_suffix` removes the suffix from the end of a string.
```

What happens if the suffix is not found (reference the prefix definition which states `If the string does not start with the prefix, the string is returned unchanged.`